workflow/rocoto/gfs_tasks.py

```diff
@@ -796,6 +796,32 @@ def postsnd(self):
 
         return task
 
+    def fbwinds(self):
+
+        atmos_prod_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_GRIB_GRID_TMPL"], {'RUN': self.cdump, 'GRID': '0p25'})
+        deps = []
+        data = f'{atmos_prod_path}/{self.cdump}.t@Hz.pgrb2.0p25.f006'
+        dep_dict = {'type': 'data', 'data': data, 'age': 120}
+        deps.append(rocoto.add_dependency(dep_dict))
+        data = f'{atmos_prod_path}/{self.cdump}.t@Hz.pgrb2.0p25.f012'
+        dep_dict = {'type': 'data', 'data': data, 'age': 120}
+        deps.append(rocoto.add_dependency(dep_dict))
+        data = f'{atmos_prod_path}/{self.cdump}.t@Hz.pgrb2.0p25.f024'
+        dep_dict = {'type': 'data', 'data': data, 'age': 120}
+        deps.append(rocoto.add_dependency(dep_dict))
+        dependencies = rocoto.create_dependency(dep=deps, dep_condition='and')
+
+        resources = self.get_resource('awips')
+
+        # TODO: It would be better to use task dependencies on the
+        # individual post jobs rather than data dependencies to avoid
+        # prematurely starting with partial files. Unfortunately, the
+        # ability to "group" post would make this more convoluted than
+        # it should be and not worth the complexity.
```

Comment:
Is this still warranted? Unless the filesystem has hung and not reporting for 2 mins, I think we are good.